src/status_im2/contexts/wallet/common/utils.cljs

```diff
@@ -8,4 +8,5 @@
 
 (defn get-derivation-path
   [number-of-accounts]
-  (str constants/path-wallet-root "/" number-of-accounts))
+  (let [path (str constants/path-wallet-root "/" number-of-accounts)]
+    (string/replace path "/" " / ")))
```

Comment:
can you double check "create-account" still works with these changes? (if it works it will redirect you to the default account page - for the moment)
otherwise we will need another function like "format-derivation-path" or something.